src/Core/Vipr.Core/CodeModel/OdcmCapabilities.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Vipr.Core.CodeModel
+{
+    public class OdcmCapabilities
+    {
+        public bool Insertable { get; set; }
```

Comment:
I don't think the OdcmCapabilties representation is mutually excusive with the originally discussed model. (You can create the original model -> parse the original model to get a OdcmCapabiltiies instance)
The current implementation reimplements the original parsing specified to items in the capabilities vocabulary and then populates the OdcmCapabilities datastructure. I don't think that's the right approach. :(